src/optimized.tsx

```diff
@@ -1,28 +1,30 @@
 import * as React from 'react';
 
-import { getComponent, getComponentFromCache, isCached } from './map';
+import { cache, getComponent } from './map';
 import { mapLoadable } from './bundler';
 
 type Props<T> = T & {
   screenName: string;
 };
 
 const Suspender = React.forwardRef(function <T extends {}>({ screenName, ...rest }: Props<T>, ref: React.Ref<any>) {
-  const Component = isCached(screenName) ? getComponentFromCache(screenName).component : null;
+  const isCached = cache.has(screenName);
 
-  if(!Component) {
+  if(!isCached) {
     throw getComponent(screenName);
   }
 
+ const { component: Component } = cache.get(screenName)!;
```

Comment:
Is this really needs to be changed?..

Reply:
for me, it's easier to read instead of this 
```const Component = isCached(screenName) ? getComponentFromCache(screenName).component : null;```